packages/terra-clinical-data-grid/src/orion-fusion-theme/RowSelectionCell.module.scss

```
@@ -0,0 +1,9 @@
+:local {
```

@import '~terra-mixins/lib/Mixins';
```suggestion
:local {
```

---

Added 46e8de9